servatrice/migrations/servatrice_0021_to_0022.sql

```diff
@@ -0,0 +1,16 @@
+
+CREATE TABLE IF NOT EXISTS `cockatrice_audit` (
+  `id` int(7) unsigned zerofill NOT NULL auto_increment,
+  `id_server` tinyint(3) NOT NULL,
+  `name` varchar(35) NOT NULL,
+  `ip_address` varchar(255) NOT NULL,
+  `clientid` varchar(15) NOT NULL,
+  `incidentDate` datetime NOT NULL default '0000-00-00 00:00:00',
+  `action` varchar(35) NOT NULL,
+  `results` ENUM('fail', 'success') NOT NULL DEFAULT 'fail',
+  `details` varchar(255) NOT NULL,
```

Comment:
Im not sure I am a fan of this idea, requiring a decoder ring to decipher the various reasons why something failed sounds like a bad idea.  However I can drop any details in the success entries since we probably could care less of the details on a successful entry.  Plus given the majority of audit entries will most likely be a success this would reduce the size of the table's potential growth.